README.md

```diff
@@ -1,15 +1,107 @@
+<div align="center">
+
+# 🖌 vello
+
+**An experimental GPU compute-centric 2D renderer**
+
+[![Xi Zulip](https://img.shields.io/badge/Zulip-Chat-blue?logo=Zulip)](https://xi.zulipchat.com/)
+[![dependency status](https://deps.rs/repo/github/linebender/vello/status.svg)](https://deps.rs/repo/github/linebender/vello)
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](#License)
+<!-- [![Crates.io](https://img.shields.io/crates/v/vello.svg)](https://crates.io/crates/vello) -->
+<!-- [![Docs](https://docs.rs/vello/badge.svg)](https://docs.rs/vello) -->
+<!-- [![Build status](https://github.com/linebender/vello/workflows/CI/badge.svg)](https://github.com/linebender/vello/actions) -->
+
+</div>
+
+Vello is a 2d graphics middleware for [`wgpu`]. 
```

Comment:
I know the term "middleware" is common in 3D contexts, but it sounds strange to my ears otherwise, I think I'd prefer "rendering engine."